lib/generator.js

```diff
@@ -94,6 +94,10 @@ function buildOperations(spec) {
         }
       }
     }
+    const formData = method.parameters.filter(param => param.in === 'formData');
+    if (formData) {
+      operation.formData = formData
+    }
```

Comment:
Do we have any validation from any other SDK's that this is ok with them?

@laura-rodriguez 
@arvindkrishnakumar-okta 
@MikeMondragon-okta

Reply:
I'll validate this on Java SDK side.